Docs/ios-screen-sharing.md

```diff
@@ -129,3 +128,15 @@ class MyDelegate: BroadcastManagerDelegate {
 }
 BroadcastManager.shared.delegate = MyDelegate()
 ```
+
+#### Custom Identifiers
+
+By default, the app group identifier and broadcast extension bundle identifier must follow the format prescribed in the setup guide. If this is not suitable for your app, you can override the defaults:
```

Comment:
rather than say "the format prescribed in the setup guide", just describe it directly here. the whole thing could be a lot more succinct

> By default the app group identifier is expected to be `group.<main-app-bundle-id>` and the broadcast extension is expected to be `<main-app-bundle-id>.broadcast`. To override these values, set `RTCAppGroupIdentifier` in your Info.plist (both broadcast extension and main application) and `RTCScreenSharingExtension` in your main app's Info.plist.